python/dgl/dataloading/labor_sampler.py

```diff
@@ -28,8 +28,8 @@
 
 class LaborSampler(BlockSampler):
     """Sampler that builds computational dependency of node representations via
-    labor sampling for multilayer GNN from
-    `(LA)yer-neigh(BOR) Sampling: Defusing Neighborhood Explosion in GNNs
+    labor sampling for multilayer GNN from the NeurIPS 2023 paper
+    `Layer-Neighbor Sampling -- Defusing Neighborhood Explosion in GNNs
     <https://arxiv.org/abs/2210.13339>`__
```

Comment:
Does there need to be underscores here?

Reply:
Yes, otherwise the hyperlink won't work.

https://docs.dgl.ai/en/1.1.x/generated/dgl.sampling.sample_labors.html